scalameter-core/src/main/scala/org/scalameter/utils/ClassPath.scala

```diff
@@ -8,51 +8,73 @@ import org.apache.commons.lang3.SystemUtils
 
 
 
-object ClassPath {
-  /** Returns the default classpath string.
+class ClassPath private (val paths: List[String]) extends Serializable {
+  /** Returns platform dependent classpath string.
    */
-  def default: String = {
-    extract(this.getClass.getClassLoader, sys.props("java.class.path"))
+  def mkString: String = {
+    val cp = paths.mkString(File.pathSeparator)
+    if (SystemUtils.IS_OS_WINDOWS) "\"" + cp + "\"" else cp
+  }
+
+  /** Prepends library to the classpath.
+   */
+  def ::(library: String) = {
```

Comment:
This is not strictly speaking a library, but a `jar` or a directory that contains classfiles.